dev-libs/weston/weston-9999.ebuild

```diff
@@ -120,7 +120,7 @@ src_configure() {
 		-Dtools=debug,info,terminal
 		-Dsimple-dmabuf-drm=auto
 		$(meson_use examples demo-clients)
-		$(usex examples -Dsimple-clients=damage,dmabuf-v4l,im,shm,touch$(usex gles2 ,dmabuf-egl,egl "") "")
+		$(usex examples -Dsimple-clients=damage,dmabuf-v4l,im,shm,touch$(usex gles2-only ,dmabuf-egl,egl "") "")
```

Comment:
I'm going to change this line now because I've realised you need to pass `-Dsimple-clients=` when you don't want any of the clients.